dts/bindings/rtc/artasie,am1805.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Linumiz
+# Author : Sri Surya <srisurya@linumiz.com>
+# PDX-License-Identifier: Apache-2.0
+
+description: ARTASIE AM1805 RTC
+
+compatible: "artasie,am1805"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+  - name: pm.yaml
+
+properties:
+  reg:
```

Comment:
IMO it's part of i2c-device.yaml and we don't need that in the consumer. Not sure!